labs/projecteuler/projecteuler.md

```diff
@@ -0,0 +1,132 @@
+# Properties in Cryptol, or How I Learned to Stop Worrying and Love SAT Solvers
+
+You have learned in the other labs that you can use Cryptol to find answers to problems you coded using the `:sat` command.  As a refresher, let's look at a simple example.  Suppose we want to find the integer that is one less than 1001 (as we said, it's very simple).  We do this by creating a function and a `property`:
+
+```
```

Comment:
The haskell mode provides some highlighting for example code blocks like this.

Reply:
Cryptol's literate system only accepts code blocks that start and end with "clean" backticks.

Reply:
That's how we can have shell commands and literate cryptol in the same file :-)